parachain/runtime/snowbridge/src/lib.rs

```diff
@@ -585,6 +585,8 @@ parameter_types! {
 	pub const MaxVoluntaryExitSize: u32 = 16;
 	pub const MaxAttestationSize: u32 = 128;
 	pub const MaxValidatorsPerCommittee: u32 = 2048;
+	pub const MaxSlotsPerHistoricalRoot: u64 = 8192;
+	pub const MaxFinalizedHeaderSlotArray: u32 = 1000;
```

Comment:
Just curios how to estimate the threshhold here? 
seems we still need to cache enough finalized checkpoints make sure we can always find the merkle root of the block roots [here](https://github.com/Snowfork/snowbridge/blob/36c99689aab88a82529931212d72a2157dd8954b/parachain/pallets/ethereum-beacon-client/src/lib.rs#L692-L701), right?

Reply:
For now, it's just an arbitrary number which can still be finetuned. That array is used for the beacon relayer, so it knows which finalized headers are already available on the parachain so we can send a block root proof for a much older block.

Reply:
That is ok. Just a little confusing about threshold in relayer with some very different value
https://github.com/Snowfork/snowbridge/blob/36c99689aab88a82529931212d72a2157dd8954b/relayer/relays/beacon/cache/cache.go#L13 

Any way, I understand these threshhold values are only for pruning the old checkpoint data which is not the main scope in this PR and we can handle it in https://linear.app/snowfork/issue/SNO-328

Reply:
Ah I see! That is a different kind of value. Let me explain the difference.

`MaxFinalizedHeaderSlotArray` is the max number of items in `FinalizedBeaconHeaderSlots`. `FinalizedBeaconHeaderSlots` holds the slots of the finalized headers we have verified and imported into the light client. The array is used for the relayer to know which finalized headers the parachain has already imported and can be used for `block_root` proofs. For instance, if the relayer goes down and starts up, the relayer won't know which finalized headers are available for proofs, so the relayer queries the parachain to get this information.

In the relayer, `FinalizedCheckpointsLimit` is the number of finalized header's block root proof tree should be kept in memory. I need to double check if I do this, but this limit can be smaller since if we clear the cache, we can download the beacon state to get this value again. The limit needs to be smaller because it is a larger structure to keep in memory, while `FinalizedBeaconHeaderSlots` is just an array of ints.

Reply:
Thanks for the detailed explaination. Yes, that make sense.